### PR TITLE
ostree-ng.sh: simplify .ks

### PR DESCRIPTION
**Pull request by teg**
Bring the kickstart closer to the embedded kicktstart.

The embedded kickstart performs a default installation, but requires you to select the partitioning. This does the same, but applies autopart.

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change

### CODE REVIEW

File: test/cases/ostree-ng.sh [hunk 1]
```diff
@@ -340,45 +340,10 @@ sudo qemu-img create -f qcow2 "${LIBVIRT_IMAGE_PATH}" 20G
...
 zerombr
 clearpart --all --initlabel --disklabel=msdos
 autopart --nohome --noswap --type=plain
-ostreesetup --nogpg --osname=rhel-edge --remote=rhel-edge --url=file:///ostree/repo --ref=${OSTREE_REF}
-poweroff
-
-%post --log=/var/log/anaconda/post-install.log --erroronfail
-
-# no sudo password for user admin
-echo -e 'admin\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```
Comment (henrywang) [inline test/cases/ostree-ng.sh:362]: We need this one for avoid sudo password when run command via ssh.

**Comment by henrywang** [comment]
@teg, test is stuck at anaconda installation. That means we need more "instructions" to make it automatically.

**Comment by atodorov** [comment]
> @teg, test is stuck at anaconda installation. That means we need more "instructions" to make it automatically.

At which screen ? That's the command which is missing.

From the old tar kickstart test in lorax composer we use the following kickstart:

```
cmdline
lang en_US.UTF-8
timezone America/New_York
keyboard us
rootpw --lock
sshkey --username root "$PUB_KEY"
bootloader --location=mbr
zerombr
clearpart --initlabel --all
autopart
# reboot is used together with --no-reboot qemu-kvm parameter, which makes the qemu-kvm
# process exit after the installation is complete and anaconda reboots the system
# (using 'poweroff' ks command just halted the machine without powering it off)
reboot

liveimg --url file:///var/lib/lorax/composer/results/$UUID/root.tar.xz
```

File: test/cases/ostree-ng.sh (hunk 1 above)
Comment (teg) [reply test/cases/ostree-ng.sh:362]: Thanks! I tried fixing this by adding a password instead.

**Comment by teg** [comment]
@henrywang @atodorov seems like I should not have dropped the `poweroff` command. Pushed with that readded.

File: osbuild-composer.spec [hunk 2]
```diff
@@ -334,6 +334,7 @@ Requires:   libvirt-daemon-driver-secret
 Requires:   libvirt-daemon-driver-storage
 Requires:   libvirt-daemon-driver-storage-disk
 Requires:   libvirt-daemon-kvm
+Requires:   lorax
```
Comment (henrywang) [inline osbuild-composer.spec:337]: @teg Looks `mkksiso` not included in RHEL release. https://src.osci.redhat.com/rpms/lorax/blob/rhel-8.4.0/f/lorax.spec#_208. It's only available for Fedora.

Comment (teg) [reply osbuild-composer.spec:337]: Yeah, this is a pain. I got this working locally by spinning up a container, but that might be overkill.

Comment (ondrejbudai) [reply osbuild-composer.spec:337]: Is it in EPEL? If so, we can just add it here: https://github.com/osbuild/osbuild-composer/blob/c2dc9d65fb71add09d5afc4f6e66991c46a7a1c0/tools/provision.sh#L11

Comment (teg) [reply osbuild-composer.spec:337]: @bcl is there a convenient way to get `mkksiso` on RHEL8? Is it in EPEL?

**Pull request closed without merging**